cms/models.py

```diff
@@ -112,6 +113,7 @@ def get_context(self, request, *args, **kwargs):
             **super().get_context(request),
             "product_cards_section_title": self.product_section_title,
             "products": self.products,
+            "default_image_path": DEFAULT_COURSE_IMG_PATH,
```

Comment:
@arslanashraf7 I looked over it, I guess, in Django we need to either define a template tag or filter if we need to achieve this as you suggested above because we need to pass the product back on the Property to calculate the result.

Reply:
I think we already have a tag for images, How feasible do you think it would be to update to support default images? https://github.com/mitodl/mitxonline/blob/5ad9aa49161765828a788e7bf926defdf5e1d8a8/cms/api.py#L102